hook-tests/014-set-motd.test

```diff
@@ -2,12 +2,15 @@
 
 set -e
 
-grep -q "Welcome to Ubuntu Core 24" etc/motd
+grep -q "Welcome to Ubuntu Core 24" usr/lib/motd.d/50-default
 
 test ! -e etc/default/motd-news
-test ! -e etc/update-motd.d/00-header
-test ! -e etc/update-motd.d/10-help-text
 test ! -e usr/lib/systemd/system/motd-news.service
 test ! -e usr/lib/systemd/system/motd-news.timer
 
-! grep -q unminimize etc/update-motd.d/*
+# Checks for empty update-motd.d - for future bases this will be a canary to
+# check if we need to leave anything new there.
+# shellcheck disable=SC2034 # read needs a variable
+if find etc/update-motd.d -mindepth 1 -maxdepth 1 | read -r var
```

Comment:
```
test -d etc/update-motd.d
test ! -s etc/update-motd.d
```

Reply:
I did a quick try but it looks like `-s` does not work for directories, or at least it give the same result for empty or non-emoty directory.

Reply:
`/bin/sh` -> `/bin/bash` maybe? I would not be surprised dash misbehaves.

Reply:
Ah no, you are right. It depends on the filesystem. It happens to work on btrfs.

Reply:
Wow, that is weird... good to know